crates/rune/src/module/function_meta.rs

```diff
@@ -111,6 +112,31 @@ impl FunctionMacroData {
     }
 }
 
+/// Runtime data for an attribute macro.
+#[derive(Clone)]
+pub struct AttributeMacroData {
+    pub(crate) item: ItemBuf,
+    pub(crate) handler: Arc<AttributeMacroHandler>,
+}
+
+impl AttributeMacroData {
+    #[inline]
+    pub(crate) fn new<F, N>(name: N, f: F) -> Self
```

Comment:
This is never used (Clippy warning), Using it would require extending the `rune::macro_` attribute since that's where it's used. It's also something we should do since it allows for registering the attribute macro using `Module::macro_meta` (the new cool way to do it).